gaftools/cli/order_gfa.py

```diff
@@ -38,32 +41,45 @@ def run_order_gfa(
 
     components = graph.all_components()
     logger.info(f"Connected components: {len(components)}")
+
     # name_comps checks the most frequent SN tag for the node in the component
     # and returns a dict of chromosome_name: {nodes...}
     components = name_comps(graph, components)
-    if not chromosome_order == "":  # user gave a list
+    # if not chromosome_order is None:  # user gave a list
+    if chromosome_order != ['']:  # user gave a list
```

Comment:
Line 22 you have `if not chromosome_order is None` and here it is `if chromosome_order != ['']`?

Reply:
Yeah, I guess a relic from the previous way it was checking that we fixed together yesterday, I'll fix that, then I don't need to check for this again later